reconcile/gql_definitions/terraform_resources/terraform_resources_namespaces.py

```diff
@@ -590,12 +592,14 @@ class NamespaceTerraformResourceSecretsManagerServiceAccountV1(
 class AssumeRoleV1(ConfiguredBaseModel):
     aws: Optional[list[str]] = Field(..., alias="AWS")
     service: Optional[list[str]] = Field(..., alias="Service")
+    federated: Optional[str] = Field(..., alias="Federated")
```

Comment:
I don't understand where this new field is being used.

Reply:
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/70140/diffs#1ab2116cf4200d3ee1d441a014b545e06e6abf46_36_47